projects/weathr.md

```diff
@@ -0,0 +1,139 @@
+---
+title: weathrly
+---
+#weathrly
+
+It's important to know what Mother Nature is going to throw at you each day -- what if you wear flip flops because it's sunny and 65 degrees in the morning and then it snows all afternoon (this is Colorado, after all).
+
+As web developers you're going to be living in the context of the web. With this next project you will be interacting with a web-api. Ultimately this is going to be a preview for what the real world is going to look like for most of you. Data is important and thats what the API provides. As a web-developer you will be giving value to that data by visualizing and organizing that data for your user base.
+
+In this project you will be getting your first real interactions with ReactJS, asynchronous javascript, and managing the state of your application.
+
+## Phase one
+
+* Your application should have a welcome page that greets the user. The welcome page has an input field that asks the user for it's  current location / desired location
+
+* Once the desired location is submitted a weekly forecast overview (morning/evening for 7 days) is shown.
+
+* There should be a way for the user to change location information. Once the location information is changed the weather should reflect the desired location.
+
+* utilize sass variables and color scheme of app changes based on the weather and temp.
+
+* location persists after refresh.
+
+
+## Phase two
+
+* Display likely-hood of weather event happening
+
+* Each day should have a summary of what the weather will look like ex: `Today: Sunny Today, It's currently 42, the high will be 46`
+
```

Comment:
'Application should handle when a user searches for a location that is not found by the API. Team should research different ways to handle unfound locations by looking around on different weather and other location searching applications and implement a solution for their project that they find intuitive.'

Not sure if this makes sense, but this might be a good way to handle this?